src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -84,7 +84,7 @@ public void MercuryLoginTest_007_registerNowLinkWorks() {
     Assertion.assertEquals(expectedHeader, currentHeader);
   }
 
-  @Test(groups = "MercuryLoginTest_008")
+  @Test(groups = {"MercuryLoginTest_008", "LoginTests_002"})
   public void MercuryLoginTest_008_userIsTakenToJoinPage() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)